edgar_tool/text_search.py

```diff
@@ -478,22 +482,19 @@ def _fetch_first_page_results_number(
                 min_wait_seconds,
                 max_wait_seconds,
                 retries,
-            )(
-                lambda json_response: json_response.get("hits", {}).get("hits", 0) != 0,
-                f"No results found on first page at URL {url}, aborting...\n"
-                f"Please verify that the search/wait/retry parameters are correct and try again.",
-            )
```

Comment:
removed this because otherwise it would overlap the final `PageCheckFailedError`